sql/search_derived/mobile_search_clients_last_seen_v1/query.sql

```diff
@@ -0,0 +1,135 @@
+WITH _derived_search_cols AS (
+  SELECT
+    udf.normalize_search_engine(engine) AS short_engine,
```

Comment:
Is it possible to delay normalizing the search engine to the view? If it's not normalized in the daily table, do we need to normalize it here?

Reply:
The difference is that the normalized engine is used in the computation here to get the monthly search totals.  We could however normalize in the view and then sum up the totals per normalized engine there.  I'll try that out

Reply:
Also just noticed I didn't make the view